test/dashboard.go

```diff
@@ -0,0 +1,147 @@
+package test
+
+import (
+	"net/http"	
+	"bytes"
+	"net/http/cookiejar"
+	"fmt"	
+	"io/ioutil"
+	"encoding/json"
+)
+
+var sessionCookie []*http.Cookie
+var jar *cookiejar.Jar
+var runId string
+var postDataClient  *http.Client
+
+
+func loginDashboard() (*http.Client) {
+		
+	sessionCookie  = nil			
+	jar, _ := cookiejar.New(nil)
+	
+	client := &http.Client{
+		Jar: jar,
+	}
+	
+	var jsonStr = []byte(`{"username":"tester","password":"nimbledcstester"}`)	
```

Comment:
Remove actual Array IP address and credentials with dummy string since this is public repo.